global.json

```diff
@@ -1,6 +1,7 @@
 {
-	"sdk": {
-		"version": "8.0.100-preview.7.23376.3",
-		"rollForward": "latestMajor"
-	}
+  "sdk": {
+    "version": "6.0.400",
```

Comment:
I understand not wanting to use the preview version, but why go back to 6.x.x? Maybe we could bump this to `7.0.400` unless there's a specific issue with that?

Reply:
I accidentally checked in the 8.0 preview yesterday. I agree that 7.0 makes sense, but I want to do a straight revert, and fast-follow with the upgrade to 7.0